src/app/page.tsx

```diff
@@ -24,7 +25,7 @@ export default async function Home() {
           2024. Excuse minor innacuracies as we tighten up the code.
         </p>
       </div>
-      <div className="grid w-full grid-cols-2 gap-2 md:grid-cols-6 md:gap-4 xl:grid-cols-5">
+      <div className="grid h-full w-full grid-cols-2 gap-4 md:grid-cols-6 xl:grid-cols-6">
```

Comment:
btw `h-full` is what caused the previous bugs in Safari, we should double check how it looks

Reply:
nice